client/src/components/Sidebar/menu/SubMenuCloseButton.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { createPortal } from 'react-dom';
+
+import Icon from '../../Icon/Icon';
+import { gettext } from '../../../utils/gettext';
+
+export default function SubMenuCloseButton({isVisible, dispatch}) {
+  if (!isVisible) {
+    return null;
+  }
+  return createPortal(
+    <button
+      type="button"
+      onClick={
+        () => dispatch({
+          type: 'set-navigation-path',
+          path: '',
```

Comment:
This works well for common use cases where we only have one submenu level at max. Technically, you can have multiple levels of submenu, but that's not something we officially support at the moment. There are some tweaks that need to be done, such as making sure the "active path" styles are correct in those scenarios.

If we want to support submenu nesting, this should set the path to the parent path instead of just the root path (an empty string). I tried looking into this but couldn't get it to work by passing `state.navigationPath.split('.').slice(0, -1).join('.')` for some reason.